### DISCUSSION TITLE
Allow us to delete a project (not just close it)

### DISCUSSION THREAD

**Discussion by lloydjatkinson** [community/community#40879] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

As title. When learning how to use Projects I've already created a few projects I'd like to get rid of.

**Comment by Trinhan87** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> As title. When learning how to use Projects I've already created a few projects I'd like to get rid of.

**Comment by mattcosta7** [comment, marked as answer]
@lloydjatkinson You can delete projects you're the admin of already in the project's settings menu

https://user-images.githubusercontent.com/8616962/206034341-294fea4c-043b-4229-9143-fac2bcdbe1e2.mov
